### PR TITLE
Offline build fixes

### PR DESCRIPTION
**Pull request by jan-kiszka**
Partially using #34. Can also be rebased over that when it is merged.

### CODE REVIEW

File: classes/dpkg-bazel.bbclass [hunk 1]
```diff
@@ -27,9 +27,16 @@ XZ_DEP = "xz-utils"
...
+    deb_dl_dir_import "${WORKDIR}/rootfs" ${HOST_BASE_DISTRO}-${BASE_DISTRO_CODENAME}
+
+    deb_dir="/var/cache/apt/archives"
+    ext_root="${PP}/rootfs"
+    ext_deb_dir="${ext_root}${deb_dir}"
+
     E="${@ isar_export_proxies(d)}"
     schroot -d / -c ${SBUILD_CHROOT} -u root -- sh <<EOF
         set -e
+        cp -n --no-preserve=owner ${ext_deb_dir}/*.deb -t ${deb_dir}/ || :
```
Comment (fmoessbauer) [inline classes/dpkg-bazel.bbclass:39]: can we please use hard-links here. These packages are not small (esp. the dbgsym variants) and locally I already ran out of space due to all the copies we make during the build.

File: classes/dpkg-bazel.bbclass [hunk 2]
```diff
@@ -44,7 +51,9 @@ bazel_fetch() {
         ./debian/rules local
         [ -f "$QUILT_PATCHES/series" ] && quilt pop -a
         rm -rf .pc
+        cp -n --no-preserve=owner ${deb_dir}/*.deb -t ${ext_deb_dir}/ || :
```
Comment (fmoessbauer) [inline classes/dpkg-bazel.bbclass:54]: Same here

**Comment by fmoessbauer** [comment]
The rest looks fine. Actually, I prefer to merge this series first and then rebase #34

File: classes/dpkg-bazel.bbclass (hunk 1 above)
Comment (jan-kiszka) [reply classes/dpkg-bazel.bbclass:39]: Discuss this with Isar. This here is just a copy of what upstream does.

Comment (fmoessbauer) [reply classes/dpkg-bazel.bbclass:39]: Ok. I'll do. Then I'll approve this and you can merge it.

**Review by fmoessbauer — APPROVED** [review]

**Pull request merged**